docs/admin_setup/Setting-up-access-to-Terra-staging-area.md

```diff
@@ -1,11 +1,3 @@
----
```

Comment:
Why are you deleting the yaml header?

Other than that, LGTM 👍

Reply:
I don't know how it works yet, i'll try to merge it and see the default behaviour. Will put it back if needed.